routes/index.js

```diff
@@ -9,61 +9,64 @@ var fs = require('file-system');
 var officegen = require('officegen');
 var passportFunction = require('../lib/passport');
 var smtp = require('../lib/smtp');
+var Language = require('../models/language')
+var copyrightLists = require('../lib/copyrightLists')
```

Comment:
let's rename this to copyrightValues because technically it isn't a list. You could call it copyrightTypes too.